test/Unit/phpunit.xml

```diff
@@ -1,11 +1,13 @@
-<?xml version="1.0"?>
```

Comment:
Also updating `phpunit.xml`.

For reference, see https://github.com/localheinz/repository/blob/80d0ee7a330ba2a7fcc62ea4c82bc0baa1e9a15d/test/Unit/phpunit.xml.